cli/src/commands/mod.rs

```diff
@@ -27,10 +29,20 @@ impl From<ExitCode> for CommandValue {
 pub type CommandResult = anyhow::Result<CommandValue>;
 
 /// Unique exit code values.
+#[derive(Copy, Clone)]
 pub enum ExitCode {
     Ok = 0,
+    Generic = 1,
     NotAuthenticated = 10,
     AuthenticationFailure = 11,
     PackageNotFound = 12,
     SetThresholdsFailure = 13,
+    FailedThresholds = 100,
+}
+
+impl ExitCode {
+    /// Terminate the application with this exit code.
+    pub fn exit(&self) -> ! {
+        process::exit(*self as i32);
+    }
 }
```

Comment:
I think this method provides a nice abstraction to ensure that we never accidentally exit the application with an exit code already used for anything else. This way a simple grep for `process::exit` shows locations of potential violators.